module/zfs/dmu.c

```diff
@@ -778,7 +780,7 @@ dmu_free_long_range_impl(objset_t *os, dnode_t *dn, uint64_t offset,
 {
 	uint64_t object_size;
 	int err;
-	uint64_t dirty_frees_threshold;
+	uint64_t dirty_frees_threshold, l1blks = 0;
```

Comment:
cstyle recommends declaring initialized variables on their own line, but in this case I don't think initializing `l1blks` is needed.  And it's a little misleading, because we don't re-initialize it each time through the loop, but we also don't reuse the value (it gets reset by get_next_chunk())... so I guess actually we should declare `l1blks` inside the loop.